includes/api/class-edd-api.php

```diff
@@ -101,6 +137,15 @@ class EDD_API {
 	 * @since 1.5
 	 */
 	public function __construct() {
+
+		$this->versions = array(
+			'v1' => 'EDD_API_V1',
+		);
+
+		foreach( $this->versions as $version => $class ) {
+			require_once EDD_PLUGIN_DIR . 'includes/api/class-edd-api-' . $version . '.php';
```

Comment:
This should do a file_exists before trying this. Bad things happen if hosts delete files or if we mess up for a release.